mkt/reviewers/forms.py

```diff
@@ -26,8 +25,30 @@
     STATUS_CHOICES.append((mkt.STATUS_CHOICES_API[status],
                            mkt.STATUS_CHOICES[status]))
 
+MODERATE_ACTION_FILTERS = (('', ''), ('approved', _lazy(u'Approved reviews')),
+                           ('deleted', _lazy(u'Deleted reviews')))
 
-log = logging.getLogger('z.reviewers.forms')
+MODERATE_ACTION_DICT = dict(approved=mkt.LOG.APPROVE_REVIEW,
+                            deleted=mkt.LOG.DELETE_REVIEW)
```

Comment:
just doing an actual dict `{approved:mkt.LOG.APPROVE_REVIEW...}` is faster and generally better :tm: